docs/reference/glossary-of-terms.md

```diff
@@ -85,6 +88,9 @@ A network of real-world entities (i.e., objects, events, situations, and concept
 #### Label
 Usually refers to a human-readable label corresponding to the `rdfs:label` [predicate](#predicate). Labels are typically unique per ontology. In OBO Format and in the bio-ontology literature, labels are sometimes called [Names](#Name). Sometimes in the machine learning literature, and in databases such as Neo4J, "label" actually refers to a [Category](#Category).
 
+#### Lutra
+[Lutra](https://ottr.xyz/#Lutra) is the open source reference implementation of the OTTR templating language.
+
 #### Mapping
 See [SSSOM]('SSSOM')
```

Comment:
SSSOM term on this page does not provide a definition for "Mapping".

Reply:
lol copied over from OAK - told myself I needed to fix this but forgot